Makefile

```diff
@@ -3,6 +3,9 @@
 
 install: 
 	bash ./deploy-all.sh
+
```

Comment:
For readability: rename the deploy-all.sh script to deploy-test-pods.sh (or similar), remove the "bash deploy-partner-pods.sh" line. Call both scripts in "install section" of Makefile. Create a new Makefile section for deploying test pods.

Reply:
I fixed this

Reply:
this removes the ability to do a simple #make install to setup everything.
can we have two options ? make install all and make install partners